packages/config/yarn.lock

```diff
@@ -0,0 +1,645 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Wierd yarn generated this

Reply:
You will want to remove this and run yarn again from root btw

Reply:
oh right I forgot to add the package to "workspaces"